README.md

```diff
@@ -27,6 +27,7 @@ Before working with the Jekyll code, you will need to install Jekyll:
     $ gem install jekyll
     $ gem install jekyll-redirect-from
     $ gem install html-proofer
+    $ gem install activesupport -v 4.2.7
```

Comment:
Yes, I had the same problem @cschneider had (on Ubuntu) about 2 weeks ago, and had to install the older version of activesupport.